packages/ui/src/components/NavigationMenu/SNavigationMenuItem.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+export default { name: 'SNavigationMenuItem' }
+</script>
+
+<script setup lang="ts">
+import { computed, onUnmounted } from 'vue'
+import { useMenuApi, useSubmenuApi } from '@/components/NavigationMenu/api'
+import SNavigationMenuItemBody from '@/components/NavigationMenu/SNavigationMenutemBody.vue'
+
+const props = withDefaults(
+  defineProps<{
+    index: string
+  }>(),
+  {},
+)
+
+const menuApi = useMenuApi()
+
+const isSelected = computed(() => menuApi.active.value === props.index)
+const isMenuCollapsed = computed(() => menuApi.collapsed.value)
+
+function handleClick() {
+  if (menuApi) {
+    menuApi.select(props.index)
+  }
```

Comment:
`useMenuApi()` uses `forceInject()`, so `menuApi` always exists. `if` check is unnecessary.